recipes/directx-headers/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.711.3-preview":
```

Comment:
I can drop it. It was mostly for testing purposes since there were changes in the build-system I wanted to verify would work with the upcoming version.

Reply:
Done.